config/default.json

```diff
@@ -3,11 +3,13 @@
     "chain": "mainnet",
     "displayName": "Ethereum",
     "symbol": "ETH",
+    "id": "ethereum",
```

Comment:
@patidarmanoj10 what is the id for Ethereum Classic?

An environment var shall be added to `custom-environment-variables.json` so it can be customized for the ETC bot.

Reply:
@gabmontes   id for ethereum classic is  ethereum-classic

Reply:
@patidarmanoj10 thanks. Can you please update `custom-environment-variables.json` so that id can be customized?

Reply:
@gabmontes  I thought this is being read from config files only.  I didnt find code which reading vairables from env variables.  Do you want me to add below lines as is in customer-environment-variable.json 

"eth": {
    "chain": "ETH_CHAIN",
    "displayName": "ETH_DISPLAY_NAME",
    "symbol": "ETH_SYMBOL",
     "id": "COINCAPIDETH"
    "wsUrl": "ETH_WS_URL",
    "wsTimeToLive": "ETH_WS_TIME_TO_LIVE"
  },
  "coincap": {
    "retries": "COINCAP_RETRIES",
    "baseUrl": "BASEURL"
  }

Reply:
Yes, please. Adding `"id": "COINCAP_ID_ETH"` to the `customer-environment-variable.json` will do it. More info here: https://github.com/lorenwest/node-config/wiki/Environment-Variables#custom-environment-variables.

Reply:
@gabmontes  done.